spec/models/conference_spec.rb

```diff
@@ -3,10 +3,15 @@
 RSpec.describe Conference do
   it_behaves_like 'HasSeason'
 
-  it { is_expected.to have_many(:attendances) }
-  it { is_expected.to have_many(:attendees) }
-  it { is_expected.to validate_presence_of(:name) }
-  it { is_expected.to validate_presence_of(:round) }
+  describe 'dependencies' do
+    subject { FactoryGirl.build(:conference) }
+
+    it { is_expected.to have_many(:attendances) }
+    it { is_expected.to have_many(:attendees) }
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:round) }
+    # The custom date validator is tested in test 'chronology', below
+  end
```

Comment:
It didn't work without the wrapper. I also tested the opposite:
`it { is_expected.not_to validate_presence_of(:name) }` and that fails. 👍  
Still not sure if this is the proper solution.